homeassistant/helpers/service.py

```diff
@@ -934,28 +939,27 @@ async def _handle_entity_call(
 
     task: asyncio.Future[ServiceResponse] | None
     if isinstance(func, str):
```

Comment:
We avoid the expensive iscoroutine inspection by passing the hassjob to async_run_hass_job. func can never be a coroutine only a coroutinefunction, str, or callable